catalog.yml

```diff
@@ -153,3 +153,7 @@
 - repo_url: https://github.com/greenelab/annorxiver_manuscript
   html_url: https://greenelab.github.io/annorxiver_manuscript/
   preprint_citation: doi:10.1101/2021.03.04.433874
+- repo_url: https://github.com/lotusnprod/rootstock
```

Comment:
The general convention is to name the repo based on the manuscript, for example naming the repo `lotus-manuscript`. This is entirely up to you, so happy to add to catalog as `lotusnprod/rootstock`, but if you were going to rename the repo, probably makes sense to do that before adding to catalog.

Reply:
We have multiple projects in the lotunprod already so that way would be fine, if ok for you! If according to the general convention you would suggest creating a lotusnprod/manuscript instead of rootstock, no problems, I can do it!

Please advise and I'll push accordingly

Reply:
removed the thumbnail url accordingly

Reply:
I am suggesting to rename the current repository to `something-manuscript`. I guess `manuscript` could also work if you think @lotusnprod will never have another manuscript.

`rootstock` refers to that repository being a template of sorts. It is okay to keep the name as `rootstock`, just not the convention. Repo name changes are slightly disruptive to Manubut, so in general the sooner the better. Ultimately your call.